sources/packages/api/src/services/application/application.service.ts

```diff
@@ -916,6 +920,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
       .innerJoinAndSelect("student.user", "user")
       .innerJoinAndSelect("application.offering", "offering")
       .innerJoinAndSelect("offering.educationProgram", "educationProgram")
+      .innerJoinAndSelect("application.coeDeniedReason", "coeDeniedReason")
```

Comment:
Is it really supposed to be an inner join?
It seems that it is a left join since the reason will be something to be added later, does it make sense or am I missing something?

Reply:
Even I think it has to be left-join unless there is a specific reason to use inner-join here.